test/test_torch.py

```diff
@@ -115,6 +115,39 @@ def make_contiguous_slice(size, dtype) -> torch.Tensor:
         def test_dir(self):
             dir(torch)
 
+        def test_deterministic_flag(self):
+            for deterministic in [True, False]:
+                for error_level in [0, 1, 2]:
+                    with torch.experimental.flags(deterministic=deterministic, deterministic_error_level=error_level):
+                        self.assertEqual(deterministic, torch.experimental.deterministic)
+                        self.assertEqual(error_level, torch.experimental.deterministic_error_level)
+
+            with torch.experimental.flags(deterministic=True, deterministic_error_level=0):
+                # Should not raise or warn
+                torch.experimental.alert_not_deterministic("caller")
+
+            with torch.experimental.flags(deterministic=True, deterministic_error_level=1):
```

Comment:
Magic numbers for the error level doesn't seem like very good API.  More idiomatic Python API is to use some sort of string like "warn" or "error" to switch between these cases.